### ISSUE TITLE
Agents App: Screen reader mode should come in but does not

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#309076]
Screen reader mode is not being detected/activated in the Agents app. When a user has a screen reader running, the Agents app should pick up screen reader mode but currently it does not come through.

**Comment by meganrogge** [comment]
@rperez030 @accesswatch @jooyoungseo I cannot reproduce. When I enable screen reader mode in the main window and open the agents app, it carries over. Pls lmk if you are not experiencing that.
